apps/frontend/core-dapp/src/features/testnet-onboarding/OnboardUserBanner.tsx

```diff
@@ -23,7 +23,7 @@ const OnboardUserBanner: React.FC = () => (
   <StyledText as="p" variant="text-regular-base" background="success" color="white" py={6}>
     Your testnet funds have arrived!{' '}
     <span>
-      <CustomLink href={Routes.Deposit}>Deposit now</CustomLink>
+      <CustomLink href={Routes.DepositBeforeLaunch}>Deposit now</CustomLink>
```

Comment:
Everything should go to the actual Deposit page now, wdyt?
```suggestion
      <CustomLink href={Routes.Deposit}>Deposit now</CustomLink>
```

Reply:
I just used the IDE to rename everything, do we even use this page? Haha

Reply:
I don't think we need to keep DepositBeforeLaunch and WithdrawBeforeLaunch routes and pages, I just checked website and nowhere we're pointing there so might as well remove it and speed up our build time lol

Reply:
Agree with removing this page, check [this comment](https://github.com/prepo-io/prepo-monorepo/pull/1174#discussion_r1165028403) for why not remove DepositBeforeLaunch and WithdrawBeforeLaunch